spec/cypress/e2e/annotations_overview_spec.cy.js

```diff
@@ -0,0 +1,240 @@
+      { medium: this.medium2, annotation: this.annotation2 },
+      { medium: this.medium3, annotation: this.annotation3 },
+      { medium: this.medium3, annotation: this.annotation4 },
+    ].forEach((test, i) => {
+      cy.visit("/annotations/overview");
+      cy.getBySelector(CARD_SELECTOR).eq(i).as("card");
+      cy.get("@card").parents(".accordion-collapse").siblings(".accordion-header").click();
+      cy.get("@card").click();
+
+      cy.url().should("contain", `/media/${test.medium.id}`);
+      let timestamp = `0:00:${test.annotation.timestamp.seconds}`;
+      cy.getBySelector("current-time").should("contain", timestamp);
+      cy.getBySelector("annotation-comment").should("contain", test.annotation.comment);
+    });
+  });
+});
+
+describe("Student annotation card (shared with teacher)", () => {
+  beforeEach(function () {
+    createAnnotationScenario(this, "teacher");
+
+    cy.then(() => {
+      FactoryBot.create("annotation", "with_text", "shared_with_teacher",
+        { medium_id: this.medium1.id, user_id: this.genericUser.id }).as("annotation1FromStudent");
+      FactoryBot.create("annotation", "with_text", "shared_with_teacher",
+        { medium_id: this.medium1.id, user_id: this.genericUser.id }).as("annotation2FromStudent");
+      FactoryBot.create("annotation", "with_text", "shared_with_teacher",
+        { medium_id: this.medium3.id, user_id: this.genericUser.id }).as("annotation3FromStudent");
+    });
+  });
+
+  it("has border according to annotation *category*, not its color", function () {
+    const colorMap = {
+      note: "#f78f19",
+      content: "#A333C8",
+      presentation: "#2185D0",
+      mistake: "#fc1461",
+    };
+    cy.visit("/annotations/overview");
+    [this.annotation1FromStudent, this.annotation2FromStudent, this.annotation3FromStudent]
+      .forEach((annotation, i) => {
+        cy.getBySelector(CARD_SELECTOR).eq(i).as("card");
+        const colorExpected = hexToRgb(colorMap[annotation.category]);
+        cy.get("@card").should("have.css", "border-color", colorExpected);
+      });
+  });
+
+  it("redirects to medium *feedback* video when clicked", function () {
+    [
+      { medium: this.medium1, annotation: this.annotation1FromStudent },
+      { medium: this.medium1, annotation: this.annotation2FromStudent },
+      { medium: this.medium3, annotation: this.annotation3FromStudent },
+    ].forEach((test, i) => {
+      cy.visit("/annotations/overview");
+      cy.getBySelector(CARD_SELECTOR).eq(i).as("card");
+      cy.get("@card").parents(".accordion-collapse").siblings(".accordion-header").click();
+      cy.get("@card").click();
+
+      cy.url().should("contain", `/media/${test.medium.id}/feedback`);
+      let timestamp = `0:00:${test.annotation.timestamp.seconds}`;
+      cy.getBySelector("current-time").should("contain", timestamp);
+      cy.getBySelector("annotation-comment").should("contain", test.annotation.comment);
```

Comment:
This line fails in my setup as well.
![Screenshot 2024-08-12 113909](https://github.com/user-attachments/assets/f6386887-55a3-4d2b-822c-11754db32559)